tests/Integration/RequestIntegrationTest.php

```diff
@@ -442,4 +443,22 @@ private function getStreamMock(int $size, string $content = ''): StreamInterface
 
         return $stream;
     }
+
+    private function createRequestFactory(ServerRequestInterface $request): RequestFactory
+    {
+        return new class($request) implements RequestFactory {
+            /** @var ServerRequestInterface */
```

Comment:
:pray: format the DocBlock so that it's multiline to keep consistency with the rest of the codebase

Reply:
There's a [fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/src/Fixer/Phpdoc/PhpdocLineSpanFixer.php) for that.

Reply:
Nice to know, would you be so kind to add it to the PHPCS config to avoid further issues in the future?